Gordon360/Services/HousingService.cs

```diff
@@ -93,28 +93,26 @@ public bool RemoveHousingAdmin(string id)
         /// Calls a stored procedure that tries to get the id of an the application that a given user is 
         /// applicant on for a given session
         /// </summary>
-        /// <param name="studentId"> The id to look for </param>
+        /// <param name="username"> The student username to look for </param>
         /// /// <param name="sess_cde"> Session for which the application would be </param>
         /// <returns> 
         /// The id of the application or 
         /// null if the user is not on an application for that session 
         /// </returns>
-        public int? GetApplicationID(string studentId, string sess_cde)
+        public int? GetApplicationID(string username, string sess_cde)
         {
             IEnumerable<ApartmentAppIDViewModel> idResult = null;
 
-            var idParam = new SqlParameter("@STUDENT_ID", studentId);
-            var sessionParam = new SqlParameter("@SESS_CDE", sess_cde);
-            
+            SqlParameter idParam = new SqlParameter("@STUDENT_ID", username);
```

Comment:
This change, and others like it, mean that we should change the corresponding attributes in tables and params in SPs in the database. from ID to username

Reply:
Correct, but not urgent. 

Let's focus on getting the API code reviewed and merged, and then go update the DB to match so we can do one EDMX update without needing to merging it all over the place again.

The features are still testable and runnable even if the column names haven't been updated yet, because the variable type will remain unchanged regardless